src/main/java/com/woowacamp/soolsool/core/cart/service/CartService.java

```diff
@@ -34,7 +34,9 @@ public void saveCartItem(final Long memberId, final CartItemSaveRequest request)
         final Cart cart = new Cart(memberId, findAllByMemberIdOrderByCreatedAtDesc(memberId));
         cart.addCartItem(newCartItem);
 
-        cartItemRepository.save(newCartItem);
+        newCartItem = cartItemRepository.save(newCartItem);
+
+        return newCartItem.getId();
```

Comment:
```suggestion
        return cartItemRepository.save(newCartItem).getId();
```

굳이 final을 없애지 않고 바로 반환해도 좋을 것 같아요.

final을 강제하다보니 어디서 변하는지 컴파일 단계에서 알 수 있어서 좋네요